ml_swissknife/crfm_utils.py

```diff
@@ -73,6 +136,9 @@ def crfm_completion(
                     top_p=decoding_args.top_p,
                     presence_penalty=decoding_args.presence_penalty,
                     frequency_penalty=decoding_args.frequency_penalty,
+                    top_k_per_token=1
```

Comment:
minor: when black splits up "single" line if / else it actually takes more space and is less readable than having a variable at the top for that